src/Backend/Modules/MediaLibrary/Js/MediaLibraryHelper.js

```diff
@@ -502,6 +502,22 @@ jsBackend.mediaLibraryHelper.group = {
     })
   },
 
+  getMediaItemForId: function(mediaItemId) {
+    var foundMediaItem = false
+
+    $.each(media, function(index, mediaFolder) {
+      $.each(mediaFolder, function(index, mediaItem) {
+        if (mediaItem.id === mediaItemId) {
+          foundMediaItem = mediaItem
+
+          return false
+        }
+      })
+    })
```

Comment:
before this line you should check if `foundMediaItem` is not false. If it isn't this `.each` should be broken off as well.

Reply:
@StijnVrolijk how? I don't see it why it would break